deeposlandia/datagen.py

```diff
@@ -0,0 +1,129 @@
+"""Main method to generate new datasets
+
+Example of program calls:
+
+- generate 64*64 pixel images from Shapes dataset, 10000 images in the training set, 100 in the
+validation set, 1000 in the testing set:
+```
```

Comment:
the docstring python syntax is often written in ReST, [Restructured
Text](http://docutils.sourceforge.net/rst.html) instead of markdow.

See http://docutils.sourceforge.net/docs/user/rst/quickstart.html

The ReST format
to write snippet of code can be:

```
The following snippet::

    import sys
    sys.prefix
```

with two `:` and at least four indentation whitespaces.

And if you want to inline it, you have to use two ticks such as:

```
you can call the function ``foobar``.
```

Reply:
Ok, modified.